### PR TITLE
Feature/papertrail monitoring

### PR DESCRIPTION
**Pull request by flibbertigibbet**
* Send log files to Papertrail
* Use pre-built OTP graph for production deployment, to get around 15-minute AMI build timeout

### CODE REVIEW

File: deployment/ansible/roles/cac-tripplanner.otp-data/tasks/main.yml [hunk 1]
```diff
@@ -25,12 +25,17 @@
...
 
 - name: Build OTP Graph
   command: /usr/bin/java -Xmx{{ otp_process_mem }} -jar {{ otp_jarfile }} --build {{ otp_data_dir }}
   args:
     chdir: "{{ otp_bin_dir }}"
+  when: develop or test
+
+- name: Copy Built OTP Graph to Host
+  fetch: src={{ otp_data_dir }}/Graph.obj dest=../../otp_data/ fail_on_missing=yes flat=yes
+  when: develop
```
Comment (kshepard) [inline deployment/ansible/roles/cac-tripplanner.otp-data/tasks/main.yml]: Good idea.

File: deployment/ansible/group_vars/development_template [hunk 2]
```diff
@@ -6,6 +6,10 @@ production: false
 # open trip planner vars
 otp_process_mem: 2G
 
+# papertrail logging
+papertrail_host: 'SUBDOMAIN_FOR_PAPERTRAIL_ACCOUNT'
+papertrail_port: 'PORT_FOR_PAPERTRAIL_ACCOUNT'
+
```
Comment (kshepard) [inline deployment/ansible/group_vars/development_template]: Do we need a boundary_token in this file too?

**Comment by kshepard** [comment]
+1, glad Boundary was easy to set up.

File: deployment/ansible/group_vars/development_template (hunk 2 above)
Comment (flibbertigibbet) [inline deployment/ansible/group_vars/development_template]: They're both only set up for production, so I might as well remove papertrail from this template.

**Pull request merged**